rust-connectors/common/src/opt.rs

```diff
@@ -359,8 +359,9 @@ impl CommonConnectorOpt {
             Err(_) => {
                 let admin = fluvio.admin().await;
 
-                let smartmodule_spec_list =
-                    &admin.list::<SmartModuleSpec, _>(vec![name.into()]).await?;
+                let smartmodule_spec_list = &admin
```

Comment:
this requires a boolean now, we could use `all` instead of `list` that seems to be the same

Reply:
It doesn't require a boolean. It just can't figure out the type of second generic parameter, which is filter. Btw, should we bump all connectors version if we update deps?

Reply:
With all, doesnt req type sig. filter needs type sig.  It will accept string.  

Actually, this code should use filter to optimize performance but can do in separate issue.